.github/workflows/main.yml

```diff
@@ -1,14 +1,21 @@
-name: Docker image CI
+name: MyLA Build/Release
 
 on:
   push:
     # takes muliple branch names
     branches: 
       - master
-      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*' # 2021.01.01
-  
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*' # 2021.01.x
+    tags:
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9][0-9]' # 2021.01.01
+ 
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        required: true
+        default: '2023MMDD'
```

Comment:
I think T&L have been following https://calver.org/ with the version numbering of YYYY.MINOR.MICRO, instead of YYYY.MM.DD

Reply:
I changed it as you requested, but it is the default value a branch might build as `YYYY.MINOR.MICRO` if nobody change the branch name from the UI as show below. I will see this as giving a hint to people for naming

![Screenshot 2024-01-10 at 1 52 55 PM](https://github.com/tl-its-umich-edu/my-learning-analytics/assets/8579775/9a3c22ca-612e-4fa2-9fb8-1a10eff9ff1c)